lib/habits/accounts/accounts.ex

```diff
@@ -0,0 +1,49 @@
+defmodule Habits.Accounts do
+  @moduledoc """
+  The Accounts context.
+  """
+
+  alias Habits.{Repo, Accounts.Account}
+
+  def create_account(attrs \\ %{}) do
+    if Repo.exists?(Account) do
+      {:error, "An account already exists. Please log in."}
+    else
+      %Account{}
+      |> Account.changeset(attrs)
+      |> Repo.insert()
+    end
+  end
+
+  def get_account!(id), do: Repo.get!(Account, id)
+
+  def get_by_email(email) when is_binary(email) do
+    Account
+    |> Repo.get_by(email: email)
+  end
+
+  @doc """
+  Returns a list of CheckIns-counts by week, beginning with the first week for
+  which there was a CheckIn.
+
+  We ignore the %Account() argument assuming there can be only one,
+  but we should probably still specify that.
+  """
+  def time_series_check_in_data(_account) do
```

Comment:
I renamed this method to make it more clear *what kind* of "data" it returns, since that's an extremely generic term.